specifications/timeout-warning/templates/nunjucks/template.njk

```diff
@@ -0,0 +1,28 @@
+{%- macro timeoutWarning(sessionDuration) %}
+
+<div class="govuk-timeout-warning-fallback">
+  <p>We will reset your application if you do not complete the form and press Next
+  by {{ sessionDuration }} minutes.</p>
+  <p>We do this to keep your information secure.</p>

```

Comment:
Have we run the content past Cogsy yet? I guess also the content of the Session Timed out page

Reply:
I will pass into him about the content probably tomorrow. I will let you know when it is approved.

Reply:
Now the content is being renewed (Phill Cogger sent the final content) :)